pkg/dash/dash_test.go

```diff
@@ -173,60 +169,6 @@ func TestNewRunnerLoadsValidKubeConfigFilteringNonexistent(t *testing.T) {
 	require.Equal(t, "test-context", kubeConfigEvent.Data["currentContext"].(string))
 }
 
-func TestNewRunnerUsesClusterClient(t *testing.T) {
```

Comment:
Why is these test being deleted? That wasn't clear to me?
If the watcher for CRDs are not shutting down properly I think we want to fix what ever is causing that.

Reply:
Added back this test